threadit-web/src/pages/PostSpool/PostSpool.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Button, Card, CardBody, Flex, FormControl, FormLabel, Input, Stack} from "@chakra-ui/react";
+import { PageLayout } from "../../containers/PageLayout/PageLayout";
+import { navStore } from "../../stores/NavStore";
+import { userStore } from "../../stores/UserStore";
+import SpoolAPI from "../../api/SpoolAPI";
+
+export default function PostSpool() {
+    const profile = userStore.userProfile;
+    const [title, setTitle] = React.useState('');
+    const [lockInputs, setLockInputs] = React.useState(false);
+    const [interests, setInterests] = React.useState('');
+
+
+
+    const postSpool = async () => {
+        if (profile) {
+            setLockInputs(true);
+            try {
```

Comment:
Having the user split interests with commas is not great UI. Maybe we could have it set up where once the user enters a space the interest is added as a little box. Something like this:

![image](https://user-images.githubusercontent.com/82107652/219880515-e3362c73-5027-4088-8f41-8bc0eec87b8c.png)

I'm okay with leaving it this way for now but definitely something we should change in the future.

Reply:
I like this idea, but no clue how you would even begin to set that up lol